apps/builder/app/shared/tree-utils.test.ts

```diff
@@ -38,6 +42,22 @@ const createInstance = (
   };
 };
 
+const createInstancePair = (
```

Comment:
```suggestion
const createInstanceTuple= (
```

Reply:
It's not some random tuple but specifically a pair of key/value. Also widely used term.